form/form_themes.rst

```diff
@@ -371,6 +371,65 @@ by ``row``, ``widget``, ``label``, or ``help``):
         {# the field of TaskType #}
     {% endblock %}
 
+Access data in a Form Choice
+.............................
+
+When you have a ChoiceType or an EntityType, and you want to customize the ``entry``, row or widget, you can also retrieve
+the data of the choice, but not directly in the ``form.vars.data`` of the child. See an exemple::
```

Comment:
We could improve this sentence altogether by defining what is actually the entry data, before suggesting to use `form.parent.vars.choices`, here's a possible alternative:
```rst
When you use an expanded ``ChoiceType`` and need to customize the children ``entry`` blocks,
the ``form.vars`` of entries (radio button or checkbox) may not be enough since each holds a
boolean value meaning whether a choice is selected or not.
To get the full list of choices data and values, you will need to access the ``choices`` variable
from their parent form (the ``ChoiceType`` itself) with ``form.parent.vars.choices``:
```